CHANGELOG.rst

```diff
@@ -116,7 +116,8 @@
   Example '-o xfail_strict=True'. A complete ini-options can be viewed
   by py.test --help. Thanks `@blueyed`_ and `@fengxx`_ for the PR
 
-*
+* Allow passing a custom debugger class (e.g. ``IPython.core.debugger:Pdb``
+  via ``--pdbcls``.  Thanks to `@anntzer`_ for the PR.
```

Comment:
We need to add a reference to `anntzer` down below, otherwise `linting` will fail. 

Also, we should have a closing parenthesis after `--pdbcls` here.